Fakes/Fakes/Fakes.generated.swift

```diff
@@ -1294,6 +1305,16 @@ extension StatGranularity {
         .day
     }
 }
+extension StateOfACountry {
+    /// Returns a "ready to use" type filled with fake values.
```

Comment:
Since "state" might be an ambiguous or unfamilar term to some maintainers, would you mind adding a comment along the lines of "with fake values for a state/province of a county"

Reply:
Or, is this not possible since this is generated code?

Reply:
This is generated code, so if I edit the file, the next time that someone else generates new fakes, my changes will be deleted. 👍